apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1775,17 +1775,16 @@ block_txs_list_by_hash_invalid_range(_Config) ->
     ok.
 
 naming_system_manage_name(_Config) ->
-
     {ok, PubKey} = rpc(aec_keys, pubkey, []),
-    PubKeyEnc = aec_base58c:encode(account_pubkey, PubKey),
-    Name       = <<"fooo.bar.test">>,
-    NameSalt   = 12345,
-    NameTTL    = 60000,
-    Pointers   = <<"{\"account_pubkey\":\"", PubKeyEnc/binary, "\"}">>,
-    TTL        = 10,
-    NHash      = aens_hash:name_hash(Name),
-    Fee        = 2,
-    MineReward = rpc(aec_governance, block_mine_reward, []),
+    PubKeyEnc   = aec_base58c:encode(account_pubkey, PubKey),
+    Name        = <<"詹姆斯詹姆斯.test"/utf8>>,
```

Comment:
It would be nice to have small additions in this test for checking handling of punycode in API.
One small addition I am thinking about is, after [the check that the name is present passing it as Unicode](https://github.com/aeternity/epoch/pull/743/files#diff-14dc0eab04e011248c7640cb3c1cf0deR1833), performing the same get_name call passing the punycode representation of the domain. This may be done in a following PR.

Reply:
> One small addition I am thinking about is, after the check that the name is present passing it as Unicode, performing the same get_name call passing the punycode representation of the domain.

Just a clarification: here I am not implying it shall succeed, though it would nice to have the assertion on how it shall behave - potentially commented out. (And yet, not needed for this PR.)

Reply:
> One small addition I am thinking about is, after the check that the name is present passing it as Unicode, performing the same get_name call passing the punycode representation of the domain.

Another small addition I am thinking about - not related to punycode this time - is, after the check that the name is present passing it as Unicode, performing the same get_name call passing a different Unicode representation of the same name (some Unicode strings have some "equivalent" Unicode representations e.g. composed vs. decomposed, and they do not need necessarily to fit any of the specified normalization forms while still being valid Unicode). (And yes, not needed for this PR.)